app/templates/content/suppliers-guide-g-cloud.html

```diff
@@ -0,0 +1,248 @@
+  </p>
+
+
+  <h3 id="yourinfo">
+  <span class="number">2.5 </span>Assurance checks will be made on your information</h3>
+
+  <p>
+  CCS checks the cloud services listed on the Digital Marketplace to make it easier for buyers to find and evaluate them.
+  </p><p>
+  The information you provide for each service is checked against the service definitions submitted for each service as part of the assurance process.
+  </p><p>
+  After you submit your application, it will be evaluated against a set of criteria published in the ITT pack. CCS will evaluate applications against a core set of criteria, as well as against a set of lot-specific criteria.
+  </p><p>
+  The features are assured so that:
+  </p>
+
+  <ul>
+  <li>the service is suitable for the framework</li>
+  <li>a consistent base level of information on the Digital Marketplace is given</li>
+  <li>buyers are aware of core service features</li>
+  </ul>
+
+
+  <h3 id="live">
+  <span class="number">2.6 </span>Your services will go live on the Digital Marketplace</h3>
+
+  <p>If you’re successful in your application, you’ll enter a framework agreement with G-Cloud. Your services will be made available on the Digital Marketplace as soon as the framework goes live.
+  </p><p>
+  Customers who want to buy your services must enter into a call-off contract with you and accept the terms and conditions you submitted at the time of application. The maximum length of a call-off contract is 24 months.
+  </p>
+  <p>
+  The G-Cloud framework is unlike most other frameworks because it incorporates the suppliers’ terms and conditions. In the case of any contradiction between the supplier’s service terms and conditions, and the framework terms and conditions, the framework terms will take precedence.
+  </p>
+
+  <h2 id="when-to-apply">
+  <span class="number">3. </span>When to apply</h2>
+
+  <p>A new version of the G-Cloud framework is released about every 6 months. Each version of the framework runs for 12 months so frameworks will overlap. </p>
+
+  <h3 id="new-suppliers">
+  <span class="number">3.1 </span>New suppliers</h3>
+
+  <p>
+  All cloud services on the Digital Marketplace are part of the G-Cloud framework – a catalogue of services where the basic terms of use have already been agreed between the government and suppliers. This means buyers can buy services without running a full tender process.
+  </p>
+
+  <p>
+  If you’re new to G-Cloud you can tender to supply services when a new version is published on the Official Journal of the European Union (<abbr title="Office Journal of the European Union">OJEU</abbr>). You don’t need to be based in the UK to apply to the framework, but you need to agree to the terms of the framework agreement and call-off contract, which are governed by English law.
+  </p>
+
+  <p>Follow the steps in <a href="#apply">‘How to apply’</a> to submit a new tender. </p>
```

Comment:
Is this link just a placeholder for now?  Should the quotation marks be part of the anchor text?  It looks a bit weird to me, but happy to defer to the style guide if that's what it says.